retail-managment-system/rms-ui/src/app/pages/investor-page/investor-page.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="container-fluid">
+  <br /><br />
+  <div class="row">
+    <div class="col-lg-10 offset-lg-1">
+      <div class="card-body">
+        <div class="row">
+          <div class="col-12">
```

Comment:
are you need all this div ?

Reply:
Removed card body , but row used to organize the code and separate the first row to the second